### PR TITLE
Added SG-1 Cosmos and Kava validators

### PR DESCRIPTION
**Pull request by fgallaire**

### CODE REVIEW

File: blockchains/cosmos/validators/list.json [hunk 1]
```diff
@@ -310,5 +310,11 @@
         "name": "Genesis Lab",
         "description": "Genesis Lab is a validation nodes operator in PoS networks and blockchain-focused software development company",
         "website": "https://genesislab.net"
+    },
```
Comment (vikmeup) [inline blockchains/cosmos/validators/list.json:313]: `+495 KB` - make file smaller and 256 by 256 png

Comment (fgallaire) [reply blockchains/cosmos/validators/list.json:313]: there are bigger files in the repo

Comment (fgallaire) [reply blockchains/cosmos/validators/list.json:313]: done

**Comment by catenocrypt** [comment]
Submission looks OK; awaiting fee payment.

**Comment by fgallaire** [comment]
> Submission looks OK; awaiting fee payment.

Is this a joke ? I'm working freely for you, then I have to pay you ?

**Pull request merged**